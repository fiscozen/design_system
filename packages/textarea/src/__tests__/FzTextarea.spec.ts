import { describe, it, vi } from "vitest";
import { mount } from "@vue/test-utils";
import { FzTextarea } from "..";

describe.concurrent("FzTextarea", () => {
  it("matches snaphost", async ({ expect }) => {
    const wrapper = mount(FzTextarea, {
      props: {},
      slots: {},
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
