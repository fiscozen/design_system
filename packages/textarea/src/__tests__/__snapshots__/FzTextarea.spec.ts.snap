// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FzTextarea > matches snaphost 1`] = `
"<div class="flex flex-col gap-8 items-start"><label class="text-sm"></label>
  <div class="relative"><textarea class="border-1 rounded p-10 placeholder:text-gray-300 disabled:bg-grey-100 disabled:border-grey-100 block invalid:border-semantic-error text-base border-grey-300 resize"></textarea>
    <!--v-if-->
  </div>
  <div class="text-sm flex items-center gap-6 h-20">
    <!--v-if-->
  </div>
</div>"
`;
